.github/s2n_freebsd.sh

```diff
@@ -15,7 +15,7 @@
 set -eu
 export CTEST_PARALLEL_LEVEL=$(sysctl hw.ncpu | awk '{print $2}')
 
-cmake . -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Debug
+cmake . -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Release
```

Comment:
I wonder if we should do both debug _and_ release build/tests. It's possible that some checks get compiled out in release, but silently fail.